### PR TITLE
Add keyboard navigation to spaces

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9624

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: changelog/unreleased/enhancement-add-keyboard-navigation-to-spaces [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Enhancement: Add keyboard navigation to spaces
```
Comment (fschade) [inline changelog/unreleased/enhancement-add-keyboard-navigation-to-spaces:1]: Add keyboard navigation to spaces overview

File: changelog/unreleased/enhancement-add-keyboard-navigation-to-spaces [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+We've added keyboard navigation and actions to spaces.
```
Comment (fschade) [inline changelog/unreleased/enhancement-add-keyboard-navigation-to-spaces:3]: In the spaces overview it is now possible to navigate through the individual spaces and perform actions using the keyboard

**Review by fschade — APPROVED** [review]
the code looks good, i noticed a little thing while testing, i'm just making a video. Your decision whether blocker or not

**Comment by fschade** [comment]
if i navigate to a space with the keyboard, press tab again at the point where the preview is active, the focus ends up in nirvana, after another click the title is selected.

means: 3 tabs are necessary to get from the preview to the title


https://github.com/owncloud/web/assets/49308105/fef440bc-eafa-4fbb-94c4-cc92a6aa27ff

**Comment by JanAckermann** [comment]
@fschade 

https://github.com/owncloud/web/assets/26169327/62d4a8da-c52a-4e48-83a6-1cdd1b630063


I can't reproduce it on my system (see Video). Furthermore this is not related to keyboard actions (or how it is implemented), could you please go to "All Files" change to tiles view and try to reproduce it there as well? I'd like you to open a new Issue with the video and give some background information about your system and browser

**Review by grimmoc — APPROVED** [review]

**Comment by fschade** [comment]
me moron, i focused on tab navigation, LGTM, i open a new issue for that.

**Pull request merged**